src/renderers/dom/server/ReactServerUpdateQueue.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactServerUpdateQueue
+ * @flow
+ */
+
+'use strict';
+
+var ReactUpdateQueue = require('ReactUpdateQueue');
+var Transaction = require('Transaction');
+var warning = require('warning');
+
+function warnNoop(publicInstance: ReactComponent<any, any, any>, callerName: string) {
+  if (__DEV__) {
+    var instConstructor = publicInstance.constructor;
+    warning(
+      false,
+      '%s(...): Can only update a mounting component. ' +
+      'This usually means you called %s() outside componentWillMount() on the server. ' +
+      'This is a no-op. Please check the code for the %s component.',
+      callerName,
+      callerName,
+      instConstructor && (instConstructor.displayName || instConstructor.name) || 'ReactClass'
+    );
+  }
+}
+
+/**
+ * This is the server update queue. It uses ReactUpdateQueue while mounting
+ * then switches to ReactNoopUpdateQueue when mounted
+ * @class ReactServerUpdateQueue
+ * @param {Transaction} transaction
+ */
+class ReactServerUpdateQueue {
+  /* :: transaction: Transaction; */
```

Comment:
Scratch that, the transform needed there is an ES7 transform: [syntax-class-properties](https://github.com/babel/babel.github.io/blob/master/docs/plugins/syntax-class-properties.md)